cedar-drt/fuzz/src/schemas.rs

```diff
@@ -88,13 +88,40 @@ impl<'a, T: Equiv> Equiv for &'a T {
     }
 }
 
+impl Equiv for cedar_policy_core::est::Annotations {
+    fn equiv(lhs: &Self, rhs: &Self) -> Result<(), String> {
+        Equiv::equiv(&lhs.0, &rhs.0)
+    }
+}
+
+impl Equiv for Option<cedar_policy_core::ast::Annotation> {
+    fn equiv(lhs: &Self, rhs: &Self) -> Result<(), String> {
+        match (lhs, rhs) {
+            (Some(a), Some(b)) => {
+                if a == b {
+                    return Ok(());
+                }
+            }
+            (Some(a), None) | (None, Some(a)) => {
+                if a.val.is_empty() {
+                    return Ok(());
+                }
+            }
+            (None, None) => return Ok(()),
+        };
+        Err(format!("two annotations mismatch: {lhs:?} and {rhs:?}"))
```

Comment:
```suggestion
        Err(format!("{lhs:?} and {rhs:?} are not equivalent"))
```

Reply:
The error is already wrapped in `mismatch in namespace annotations: {e}` so there's no need to repeat the description.